### ISSUE TITLE
check bibl uris for manuscripts

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#122]
Review the bibls in a provided sheet (TBD). Check the text node of bibl elements that are manuscript references against the shelfmarks matched to the URI of the ms. Also mark any bibls that are to manuscripts but do not have a ptr element pointing to a ms URI.

First make the sheet, then assign a student to review.

**Comment by wlpotter** [comment]
This should probably wait on a few other normalizations and on xml:id stability (cf. #59 )

**Comment by wlpotter** [comment]
I believe this is the spreadsheet: https://docs.google.com/spreadsheets/d/1kONUPHs0nfLoitEym1I46mGMhj3AbTtVkc7gjsU0G-o/edit#gid=258321513 and we are having Roman review these.

**Comment by wlpotter** [comment]
Check to see why the rows are marked 'question'

**Comment by wlpotter** [comment]
This work has been superseded by other normalization issues
